presenters/booking_presenter.rb

```diff
@@ -0,0 +1,52 @@
+class BookingPresenter
+  TICKETS_COUNT_LIMIT = 5
+
+  attr_reader :first_class_available_seats, :second_class_available_seats,
```

Comment:
Use methods & memoization(`||=`) instead
https://www.justinweiss.com/articles/4-simple-memoization-patterns-in-ruby-and-one-gem/